helm/observability-operator/templates/rbac.yaml

```diff
@@ -9,6 +9,7 @@ rules:
       - ""
     resources:
       - namespaces
+      - pods
```

Comment:
Why?

Reply:
Because otherwise the job's pod isn't able to list pods

Reply:
Maybe the job should have it's own rbac?

Reply:
I would also add this `"helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded` on the job resources right?

Reply:
> Maybe the job should have it's own rbac?

Is it really worth it ? 

Also I added the delete-policy helm hook.

Reply:
Ok I'll first see if the job works as intended and if it does then I'll add its own RBAC